drivers/platform/nintendo3ds/Kconfig

```diff
@@ -15,6 +15,13 @@ menuconfig NINTENDO3DS_PLATFORM_DEVICES
 
 if NINTENDO3DS_PLATFORM_DEVICES
 
+config CTR_SDHC
+	tristate "Nintendo 3DS SD Host Controller"
+	select MMC
+	default y
+	help
+	  TODO
```

Comment:
Worth filling in.

Reply:
Yeah, I'm just terrible at descriptions but will think of something to put in there